diff --git a/.gitmodules b/.gitmodules
new file mode 100644
index 00000000..689f2314
--- /dev/null
+++ ./.gitmodules
@@ -0,0 +1,6 @@
+[submodule "quotesbot"]
+	path = quotesbot
+	url = https://github.com/scrapy/quotesbot.git
+[submodule "sehuatang"]
+	path = sehuatang
+	url = git@bitbucket.org:huangyingw/sehuatang.git
diff --git a/quotesbot b/quotesbot
new file mode 160000
index 00000000..16ec3f41
--- /dev/null
+++ ./quotesbot
@@ -0,0 +1 @@
+Subproject commit 16ec3f4176db67ca9132f7e4431d7823e5e28974
diff --git a/sehuatang b/sehuatang
new file mode 160000
index 00000000..41139cb2
--- /dev/null
+++ ./sehuatang
@@ -0,0 +1 @@
+Subproject commit 41139cb26e51919ff4b4b7f130e6bc00587fd66d
