diff --git a/.gitmodules b/.gitmodules
new file mode 100644
index 00000000..7910ab2b
--- /dev/null
+++ ./.gitmodules
@@ -0,0 +1,3 @@
+[submodule "quotesbot"]
+	path = quotesbot
+	url = https://github.com/scrapy/quotesbot.git
diff --git a/quotesbot b/quotesbot
new file mode 160000
index 00000000..16ec3f41
--- /dev/null
+++ ./quotesbot
@@ -0,0 +1 @@
+Subproject commit 16ec3f4176db67ca9132f7e4431d7823e5e28974
