diff --git a/.gitmodules b/.gitmodules
new file mode 100644
index 00000000..7f38f2bd
--- /dev/null
+++ ./.gitmodules
@@ -0,0 +1,24 @@
+[submodule "quotesbot"]
+	path = quotesbot
+	url = https://github.com/scrapy/quotesbot.git
+[submodule "sehuatang"]
+	path = sehuatang
+	url = git@bitbucket.org:huangyingw/sehuatang.git
+[submodule "scrapy-examples"]
+	path = scrapy-examples
+	url = https://github.com/geekan/scrapy-examples.git
+[submodule "Scrapy-Samples"]
+	path = Scrapy-Samples
+	url = https://github.com/mjhea0/Scrapy-Samples.git
+[submodule "scrapy-tutorial"]
+	path = scrapy-tutorial
+	url = https://github.com/harrywang/scrapy-tutorial.git
+[submodule "scrapy-spider-example"]
+	path = scrapy-spider-example
+	url = https://github.com/AccordBox/scrapy-spider-example.git
+[submodule "torrents-crawler"]
+	path = torrents-crawler
+	url = https://github.com/yasoob/torrents-crawler.git
+[submodule "get_torrents"]
+	path = get_torrents
+	url = https://github.com/philippeoz/get_torrents.git
